src/__stories__/avatar-story.tsx

```diff
@@ -17,6 +17,11 @@ export default {
             options: ['undefined', 'IconStarFilled', 'IconFireRegular', 'IconBrainRegular'],
             control: {type: 'select'},
         },
+        borderColor: {
+            defaultValue: 'borderLow',
```

Comment:
I think that is not showing defaultValue by default 🤔

https://rebrand.ly/yr48gsu